ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -311,11 +360,17 @@ private void readWithProgress(
       int requestId,
       ResponseBody responseBody) throws IOException {
     Reader reader = responseBody.charStream();
+    ProgressResponseBody progressResponseBody = (ProgressResponseBody)responseBody;
```

Comment:
I don't fully understand how this works - AFAICT `readWithProgress` is only called from `onResponse`, which makes me think the entire response is already available but sending to JS is chunked into multiple events, which doesn't really make it "incremental".